08-rails/README.md

```diff
@@ -10,17 +10,21 @@
     - [Rails Templates](rails-template-setup.md)
     - [Configuring Rails to use Postgres](how-to-use-postgres.md)
     - [Live Code Intro](live-code-intro.md)
-1. [Controllers](routes-controllers.md)
+1. [Controllers and the Index Action](routes-controllers.md)
```

Comment:
As you noted the filename here isn't the current filename, renaming the markdown would be good to do after API week I think.